types/config.go

```diff
@@ -167,6 +167,8 @@ type Config struct {
 		FirebaseCredentialsPath                       string `yaml:"firebaseCredentialsPath" envconfig:"NOTIFICATIONS_FIREBASE_CRED_PATH"`
 		ValidatorBalanceDecreasedNotificationsEnabled bool   `yaml:"validatorBalanceDecreasedNotificationsEnabled" envconfig:"VALIDATOR_BALANCE_DECREASED_NOTIFICATIONS_ENABLED"`
 		PubkeyCachePath                               string `yaml:"pubkeyCachePath" envconfig:"NOTIFICATIONS_PUBKEY_CACHE_PATH"`
+		OnlineDetectionLimit                          *int   `yaml:"onlineDetectionLimit" envconfig:"ONLINE_DETECTION_LIMIT"`
```

Comment:
Even if I agree that pointer be a better choice in the first place. I would not mix the current solution with pointers.